src/components/account/create.tsx

```diff
@@ -248,8 +250,6 @@ export class Container extends React.Component<Properties, State> {
             {Boolean(this.state.displayNameError) && this.renderError(this.state.displayNameError)}
           </div>
 
-          {Boolean(this.state.error) && this.renderError(this.state.error)}
```

Comment:
I don't think this state value is necessarily an image specific error so we might have to leave this component alone for now:

`this.setState({ error: 'Creating user failed' });`